GooglePlacesAutocomplete.js

```diff
@@ -565,24 +604,11 @@ const GooglePlacesAutocomplete = React.createClass({
           automaticallyAdjustContentInsets={false}
           {...this.props}
           renderRow={this._renderRow}
+          renderFooter={this._renderPoweredLogo}
         />
       );
     }
 
-    if(this.props.enablePoweredByContainer) {
```

Comment:
I meant users should still have the ability to disable powered logo using this prop.